1209/5858/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: LNE Keyboard Device
+owner: LNE
+license: GPLv3
+site: https://github.com/ln-org
+source: https://github.com/NWIron/qmk_firmware
```

Comment:
Please update this to the new repo.

Reply:
Thanks for you time, I have updated the source link!